### PR TITLE
Bundler docs

### PR DESCRIPTION
**Pull request by jules2689**
supersedes #299

Problem:

https://twitter.com/rappmasterflash/status/851494907906670592

This copies over the entire `doc` directory to be under `bundler.io/doc/*`. This means that `issues` is no longer treated as a special snowflake, although Code of Conduct still is.

A file at `doc/path/NAME_OF_FILE.md` would be at `bundler.io/doc/path/name_of_file.html` in the end.

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -91,7 +91,7 @@
 page /\/man\/(.*)/, layout: :commands_layout
 page /\/v(.*)\/commands\.html/, layout: :commands_layout
 page /\/v(.*)\/guides\/(.*)/, layout: :md_guides_layout
-page '/issues.html', layout: :md_guides_layout
+page '/doc/contributing/issues.html', layout: :md_guides_layout
```
Comment (segiddins) [inline config.rb]: maybe should redirect the old page for compatibility?

**Comment by segiddins** [comment]
CSS here looks a bit wonky -- https://bundler-site-staging-pr-317.herokuapp.com/doc/troubleshooting.html

**Comment by jules2689** [comment]
Yea, issue with links too

File: source/conduct.html.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Maintainers, including the core team, are also subject to the anti-harassment po
 
 If anyone engages in abusive, harassing, or otherwise unacceptable behavior, including maintainers, we may take appropriate action, up to and including warning the offender, deletion of comments, removal from the project’s codebase and communication systems, and escalation to GitHub support.
 
-If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of [the core team](http://bundler.io/contributors.html) or [email the core team](//mailto:team@bundler.io) immediately.
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of [the core team](http://bundler.io/contributors.html) or [email the core team](/mailto:team@bundler.io) immediately.
```
Comment (segiddins) [inline source/conduct.html.md]: this now 404s on github

Comment (jules2689) [inline source/conduct.html.md]: yea, was a mistaken commit

**Comment by jules2689** [comment]
@segiddins all those issues have been addressed now :tada:

**Comment by jules2689** [comment]
ping @segiddins @indirect 
This should be ready to go now

**Comment by segiddins** [comment]
@bundlerbot r+

**Pull request merged**